sqld/src/database/libsql.rs

```diff
@@ -199,16 +202,29 @@ struct Connection {
 impl Connection {
     fn new(
         path: &Path,
+        extensions: Vec<PathBuf>,
         wal_hook: impl WalHook + Send + Clone + 'static,
         with_bottomless: bool,
         stats: Stats,
     ) -> anyhow::Result<Self> {
-        Ok(Self {
+        let x = Self {
             conn: open_db(path, wal_hook, with_bottomless)?,
             state: ConnectionState::initial(),
             timed_out: false,
             stats,
-        })
+        };
+
+        for ext in extensions {
+            unsafe {
+                let _guard = rusqlite::LoadExtensionGuard::new(&x.conn).unwrap();
+                x.conn
+                    .load_extension(&ext, None)
+                    .with_context(|| format!("Loading extension: {}", &ext.display()))?;
```

Comment:
The anyhow context is printed as a standalone error message, so it will be less confusing if the error message says something like "Could not load extension: {}".